DigitalLearningSolutions.Web/Controllers/Register/RegisterInternalAdminController.cs

```diff
@@ -143,13 +156,37 @@ await featureManager.IsEnabledAsync("RefactoredTrackingSystem")
                 }
             }
 
-            return RedirectToAction("Confirmation");
+            if (model.CentreSpecificEmail != null &&
+                !emailVerificationService.AccountEmailIsVerifiedForUser(userId, model.CentreSpecificEmail))
+            {
+                var userEntity = userService.GetUserById(userId);
+                var unverifiedEmails = PossibleEmailUpdateHelper.GetSingleUnverifiedEmail(model.CentreSpecificEmail);

```

Comment:
Don't think you need this. I'm pretty sure you're just wrapping the email here and unwrapping it a few lines later...